pkg/daemon/agent/flexvolume/controller.go

```diff
@@ -250,11 +250,16 @@ func (c *Controller) parseClusterName(storageClassName string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	clusterName, ok := sc.Parameters["clusterName"]
+	clusterName, ok := sc.Parameters["clusterNamespace"]
 	if !ok {
-		// Defaults to rook if not found
-		logger.Infof("clusterName not specified in the storage class %s. Defaulting to '%s'", storageClassName, cluster.DefaultClusterName)
-		return cluster.DefaultClusterName, nil
+		// Checks for older version of parameter
+		clusterName, ok := sc.Parameters["clusterName"]
+		if !ok {
+			// Defaults to rook if not found
+			logger.Infof("clusterName not specified in the storage class %s. Defaulting to '%s'", storageClassName, cluster.DefaultClusterName)
```

Comment:
this message could mention `clusterNamespace` instead of `clusterName`

Reply:
Made the changes. Will now work on the unit tests.